warmup/app.go

```diff
@@ -0,0 +1,24 @@
+// Copyright 2016 Google Inc. All rights reserved.
+// Use of this source code is governed by the Apache 2.0
+// license that can be found in the LICENSE file.
+
+package sample
+
+import (
+	"net/http"
+
+	"google.golang.org/appengine"
+	"google.golang.org/appengine/log"
+)
+
+func init() {
+	http.HandleFunc("/_ah/warmup", warmupHandler)
+}
+
+func warmupHandler(w http.ResponseWriter, r *http.Request) {
+	ctx := appengine.NewContext(r)
+
+	// Do things using ctx.
```

Comment:
This is an oblique comment. Is the "using ctx" important? It seems to be the focus, when what you really mean is something like

```
// Perform warmup tasks (eg, blah blah).
```